src/server.h

```diff
@@ -2976,6 +2976,7 @@ unsigned char *zzlFirstInRange(unsigned char *zl, zrangespec *range);
 unsigned char *zzlLastInRange(unsigned char *zl, zrangespec *range);
 unsigned long zsetLength(const robj *zobj);
 void zsetConvert(robj *zobj, int encoding);
+void zsetConvertAndExpand(robj *zobj, int encoding, unsigned long cap);
```

Comment:
Move this to the top of t_zset.c?

Reply:
zsetConvertAndExpand was changed from zsetConvert, moving it would make a huge diff...